RELEASES.md

```diff
@@ -1,3 +1,141 @@
+Version 1.28.0 (2018-08-02)
+===========================
+
+Language
+--------
+- [Stabilised the `#[repr(transparent)]` attribute.][51562] This attribute
```

Comment:
This should be spelled as "Stabili<strong>z</strong>ed"? (also: "optimization", "prioritize", "vectorize")

* We've got the header "Stabili<strong>z</strong>ed APIs"; overall 36 matches in this document
* The spelling "Stabili<strong>s</strong>ed" is only used 3 times before this PR (this PR introduced 6 "Stabili<strong>s</strong>ed")
* There's also a misspelled "Stablise" 🤔 in 1.26.0's note.

Reply:
@kennytm Should it? I'm not from the US so when I'm writing it's stabilise, optimisation, prioritise, and vectorise.

Reply:
We should try to be consistent across the release notes. There is the "Stabilized APIs" seciton, but "Stabilised XYZ" is used everywhere else.

Reply:
I think consistency is important, and while I hate to admit it, American spelling is the standard in tech <del>even though it's wrong</del>.

We should at least be using it where people might be ctrl-F-ing.

Reply:
I remember someone saying that `ize` is valid BrE spelling...

I think it is more important to be consistent with our past selves than tech in general...

Reply:
I agree, although I was mostly appealing to the standard for tech in general because we get a lot of contributors, most of whom go with that standard. So, if we're going to be consistent with our past selves, that's probably the best bet for accomplishing that.

I think that I recall some RFC also establishing American English as the standard for Rust docs, although I'm on my phone and probably won't be able to find it right now. I'm 70% I'm not imagining that.

Reply:
I found it... https://github.com/rust-lang/rfcs/pull/2412/files/9506661d53469f17456af628fe7b1fc12cb3e275#diff-67b16d9ec3131e12b4128e3027edf279

You are not imagining it: https://github.com/rust-lang/rfcs/blob/master/text/1574-more-api-documentation-conventions.md#english-1
> All documentation for the standard library is standardized on American English, with regards to spelling, grammar, and punctuation conventions.

This makes me a bit sad since I want to rebel against the American hegemony...
A strict reading of that snippet implies that it is *only* about the standard library,
but a *"in the spirit"* reading of that RFC implies that we should go with `-ize`.

Reply:
It's okay, we just have to co-author a new RFC to make `RELEASES.md` stabilise things instead of stabilize them.

(let's not actually)

Reply:
Well, you could try to sneak it into an internationalization RFC... British-vs-American English!

Reply:
To be frank if we want to be consistent with our past these should be written as (see 1.27.0 and 1.26.0 release notes):

> The `#[repr(transparent)]` attribute is now stable.

Or simply don't focus on the stabilization process

> Users can now use the `GlobalAlloc` trait and `#[global_allocator]` attribute to specify a global allocator for their program.

Reply:
Perhaps it might actually be worth creating a proper style guide for these, as an RFC or otherwise?

Reply:
I think the release team can just create a style guide for their own purposes if need be. Don't think this rises to the level of needing an RFC.

Reply:
Yeah. Added this to the release team meeting agenda.